UI/data/locale/en-US.ini

```diff
@@ -1074,6 +1074,7 @@ Basic.Settings.Output.SplitFile.Size="Split Size"
 # Screenshot
 Screenshot="Screenshot Output"
 Screenshot.SourceHotkey="Screenshot Selected Source"
+Screenshot.PreviewHotkey="Screenshot Preview"
```

Comment:
This string is not needed as there is already a "Screenshot (Preview)" string.

Reply:
I disagree. The parentheses are needed for the context menu entry but not needed here.